browser_history/generic.py

```diff
@@ -251,15 +300,15 @@ def formatted(self, output_format="csv"):
             # fetch the required formatter and call it. The formatters are instance methods
             # so no need to pass any arguments
             formatter = self._format_map[output_format]
-            return formatter()
+            return formatter(fetch_type=fetch_type)
         raise ValueError(
             f"Invalid format {output_format}. Should be one of {Outputs.formats}"
         )
 
-    def to_csv(self):
+    def to_csv(self,fetch_type = "history"):
```

Comment:
Since a `Outputs` object is used only for either history or bookmarks (correct me if I'm wrong), you could have the `fetch_type` parameter in `__init__` itself. This way, you won't have to pass it everytime for these functions.

Reply:
Ahh good idea!